gremlin-go/driver/gorillaTransporter.go

```diff
@@ -65,21 +73,30 @@ func (transporter *gorillaTransporter) Write(data []byte) (err error) {
 	}
 
 	err = transporter.connection.WriteMessage(websocket.BinaryMessage, data)
-	return err
+	return
+}
+
+func (transporter *gorillaTransporter) getAuthInfo() *AuthInfo {
+	return transporter.authInfo
 }
 
-func (transporter *gorillaTransporter) Read() (bytes []byte, err error) {
+// Read used to read data from the transporter. Opens connection if closed.
+func (transporter *gorillaTransporter) Read() ([]byte, error) {
 	if transporter.connection == nil {
-		err = transporter.Connect()
+		err := transporter.Connect()
```

Comment:
there should be some place where we say that we support websocket V13 in the gorilla configuration.

Reply:
The design references gorilla's websocket implementation which references the v13 spec of websockets.